src/curriculum.ts

```diff
@@ -0,0 +1,17 @@
+enum nivelIngles {A1,A2,B1,B2,C1,C2};
+enum nivelEstudios {Primara,Secundaria,Bachillerato,FP,Grado,Master}
```

Comment:
```suggestion
enum nivelEstudios {
  Primara = "primaria",
  Secundaria = "secundaria",
  Bachillerato = "bachillerato",
  FP = "fp",
  Grado = "grado",
  Master = "master"
}
```
Al igual que aquí, quiero que se puedan hacer comparaciones entre cadenas.

Reply:
Vale, voy a arreglarlo.